.github/workflows/ci.yml

```diff
@@ -67,6 +67,40 @@ jobs:
       - name: Run integration tests
         run: cargo test --release --locked --package integration-tests
 
+  api-tests:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+
+      - uses: "./.github/actions/init"
+
+      - name: Run tests
+        working-directory: pop-api
+        run: cargo test --release --all-features
+
+  api-integration-tests:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+
+      - uses: "./.github/actions/init"
+
+      - name: Run integration tests
+        working-directory: pop-api/integration-tests
+        run: cargo test --release
+
+      - name: Copy contract WASM files to artifacts folder
+        working-directory: pop-api/integration-tests
+        run: |
+          mkdir -p artifacts
+          cp contracts/*/target/ink/*.wasm artifacts/
+
+      - name: Archive API contract artifacts
```

Comment:
As above, interested to hear the use case for this. If the build jobs are in parallel on the same worker, fear this may just slow the overall process down for something that not many will use.

Reply:
I think right now we don't have any specific use case for the contract artifacts yet. So would be better to simply the CI script by removing this job.